packages/dai-plugin-mcd/src/schemas/pot.js

```diff
@@ -0,0 +1,68 @@
+import { fromRay } from '../utils';
+import { CHAI } from '../..';
```

Comment:
it's not appropriate to use Chai here since the values you're getting back are not literally quantities of Chai token